public/app/features/canvas/elements/windTurbine.tsx

```diff
@@ -74,11 +74,22 @@ export const windTurbineItem: CanvasElementItem<any, any> = {
 
   defaultSize: {
     width: 100,
-    height: 100,
+    height: 155,
   },
 
   getNewOptions: (options) => ({
     ...options,
+    background: {
+      color: {
+        fixed: 'transparent',
+      },
+    },
+    placement: {
+      width: 100,
+      height: 155,
+      top: 0,
+      left: 0,
+    },
```

Comment:
This is getting really close! The only thing I'm noticing is now when user switches existing element to wind turbine, drone side / front the element placement gets shifted back to the top / left (i'm guessing due to these lines here (90-91) Once that is addressed I think this is good to be merged in 💯 

https://user-images.githubusercontent.com/22381771/181860226-043bec27-72ce-4f12-a56f-7672e5680d76.mov